cdc/cdc_options.hh

```diff
@@ -49,7 +49,8 @@ std::ostream& operator<<(std::ostream& os, delta_mode);
 std::ostream& operator<<(std::ostream& os, image_mode);
 
 class options final {
-    bool _enabled = false;
+    bool _enabled = false; // Remember to update `_is_enabled_set` when manipulating it
+    bool _is_enabled_set = false;
```

Comment:
optional was good here. You can implement both `enabled()` and `is_enabled_set()` with optional.

Reply:
done